### ISSUE TITLE
error in libcurl / curlrules.h in 0.9.0 master branch

### ISSUE THREAD

**Issue by moebiussurfing** [bakercp-archive/ofxOAuth#18]
![captura de pantalla 2015-07-23 a las 4 05 38](https://cloud.githubusercontent.com/assets/2106149/8844897/7fb2a994-30f0-11e5-9df8-5ef7d3e248b9.png)

Hey again @bakercp , any idea if I can fix this error?
Xcode 6.3.2 and 10.10.4

**Comment by bakercp** [comment]
Hm.  Unfortunately I have no idea about this one.  I use ofxHTTP (with built-in oauth) for Twitter, etc these days.

**Comment by ukelady** [comment]
@bakercp I wonder if it's because libcurl is built for 32bit. I am currently trying to build 64 bit version but no luck. I got lots of undefined symbols like this:
Undefined symbols for architecture x86_64:
  "_RTMP_ConnectStream", referenced from:
      _rtmp_do in libcurl.a(libcurl_la-curl_rtmp.o)
  "_RTMP_Alloc", referenced from:
 
Did you build the static library on your own? Or where did you find the libcurl static library? Thanks!

**Comment by bakercp** [comment]
Yes, I think that's the problem.  What are you trying to access with this library (I'm about to deprecate it as soon as I release my Twitter add-on).  Everything this library can do ofxHTTP can do, without any precompiled libraries.

**Comment by ukelady** [comment]
I am trying to upload gif files to twitter and tumbler through my openframework application. Is your Twitter add-on a replacement of ofxOAuth? Do you think I can use ofxHTTP to do this as well? Thanks in advance!

**Comment by bakercp** [comment]
hey @ukelady I'll add you to the private repo.  There's an example for uploading images.

**Comment by bakercp** [comment]
see https://github.com/SAIC/ofxTwitterTools/blob/master/example_rest_basic_status_update/src/ofApp.cpp#L44-L45

**Comment by ukelady** [comment]
That's awesome!! Thanks!!

On Sat, Apr 30, 2016 at 7:54 PM, Christopher Baker <notifications@github.com

**Comment by bakercp** [comment]
No problem @ukelady at some point in the very near future, I'll be releasing ofxTwitter (I just changed the name on the advice of @danomatika, who has been beta testing w/ his students).  Feel free to contribute.  Once I release ofxTwitter, I'll deprecate ofxOAuth (though ofxHTTP can already do oAuth1+2).   See https://github.com/bakercp/ofxCloudPlatform/ for an example of an oAuth2 google client.

**Comment by ukelady** [comment]
Hi Christopher,

I've got the example project compiled! (setting up twitter app at the
moment...)  Just to let you know that it requires the ofxHTTP develop
branch.   The master branch is too outdated for ofxTwitter

On Sat, Apr 30, 2016 at 8:41 PM, Christopher Baker <notifications@github.com

**Comment by bakercp** [comment]
Yes, it definitely requires ofxHTTP develop, sorry that wasn't clear.  When I release ofxTwitter, I'll push ofxHTTP develop to master.  Glad it's working.

**Comment by awilkie1** [comment]
Has there been any progress on fixing the twitter example or a new ofxTwitter addon ??
